specs/interop/README.md

```diff
@@ -0,0 +1,81 @@
+- (HAIP) [OpenID4VC High Assurance Interoperability Profile with SD-JWT VC](https://vcstuff.github.io/oid4vc-haip-sd-jwt-vc/draft-oid4vc-haip-sd-jwt-vc.html)
+- (EBSI) [European Blockchain Standards Initiative Interop Profile](https://ec.europa.eu/digital-building-blocks/wikis/display/EBSIDOC/)
+- (MSFT) [Microsoft Supported standards](https://learn.microsoft.com/en-us/entra/verified-id/verifiable-credentials-standards)
+
+There have been recent conversations at IIW, in the OIDF, and in DIF about updating and continuing to standardize on these profiles. In the meantime, we need to deliver working software that facilitates interoperability. As such, let’s work on our own profile that leverages the best parts of existing profiles, since none exactly fit all the features we need to support.
+
+## A tbDEX Profile
+
+Our first tbDEX profile does not prevent there from being other profiles, nor does it restrict what types of interactions are permitted on tbDEX. Instead, the profile aims to promote interoperability between implementers that can be built and iterated upon.
+
+With tbDEX we identify three main parties. The parties may overlap (e.g. a tbDEX node can be a credential issuer).
+
+- **tbDEX nodes** — run to facilitate tbDEX messages, interacts with end-users through web applications, wallets, etc.
+- **Issuers** — responsible for issuing Verifiable Credentials.
+- **Wallets/Applications** — user-centric way to interact with tbDEX, store/manage/use their identities credentials, and other data.
+
+|  | Supported | Who? | Notes |
+| --- | --- | --- | --- |
+| Entity Identifiers [Required] | did:jwk, did:dht, did:web | tbDEX, Issuer, Wallets | did:dht is recommended when possible |
+| Entity Identifiers [Optional] | did:key, did:ion | tbDEX, Issuer, Wallets | -|
+| Signature Schemes | Ed25519/EdDSA, secp256k1/ES256K, secp256r1 (P-256)/ECDSA, X25519/ECDH | tbDEX, Issuer, Wallets | Sign/Verify: ed25519 with EdDSA, secp256r1 with ECDSA, secp256k1 with ES256K Encrypt/Decrypt (via Key Agreement): X25519 and secp keys with ECDH Note: worth specifying encryption algs? |
+| Entity AuthN/Z | tbDEX Signed Messages with VCs | tbDEX, Issuer, Wallets | [As outlined here](https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#signatures). |
+| VC Formats | W3C VCDM v1.1 as VC-JWT | tbDEX, Issuer, Wallets | Most widely adopted; should be used with VC-JSON-Schema instead of JSON-LD contexts. |
+| Credential Issuance | OID4VCI | Issuer, Wallets | Will need to pick an implementers draft to implement against. |
+| Credential Presentation (Option) | OID4VP using Presentation Exchange v2 | tbDEX, Issuer, Wallets | Will need to pick an implementers draft to implement against. |
+| Status | Status List 2021 | tbDEX, Issuer, Wallets | Will implement https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/#example-example-statuslist2021credential. |
+| tbDEX Messages | v1, DIF PEv2 | tbDEX, Wallets | Yet to be versioned. |
```

Comment:
```suggestion
| Entity Identifiers [Required] | did:jwk, did:dht, did:web | Nodes, Issuer, Wallets | did:dht is recommended when possible |
| Entity Identifiers [Optional] | did:key, did:ion | Nodes, Issuer, Wallets | -|
| Signature Schemes | Ed25519/EdDSA, secp256k1/ES256K, secp256r1 (P-256)/ECDSA, X25519/ECDH | Nodes, Issuer, Wallets | Sign/Verify: ed25519 with EdDSA, secp256r1 with ECDSA, secp256k1 with ES256K Encrypt/Decrypt (via Key Agreement): X25519 and secp keys with ECDH Note: worth specifying encryption algs? |
| Entity AuthN/Z | tbDEX Signed Messages with VCs | Nodes, Issuer, Wallets | [As outlined here](https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#signatures). |
| VC Formats | W3C VCDM v1.1 as VC-JWT | Nodes, Issuer, Wallets | Most widely adopted; should be used with VC-JSON-Schema instead of JSON-LD contexts. |
| Credential Issuance | OID4VCI | Issuer, Wallets | Will need to pick an implementers draft to implement against. |
| Credential Presentation (Option) | OID4VP using Presentation Exchange v2 | Nodes, Issuer, Wallets | Will need to pick an implementers draft to implement against. |
| Status | Status List 2021 | Nodes, Issuer, Wallets | Will implement https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/#example-example-statuslist2021credential. |
| tbDEX Messages | v1, DIF PEv2 | Nodes, Wallets | Yet to be versioned. |
```